packages/pixeloven-generators/src/generators.ts

```diff
@@ -2,9 +2,14 @@ import { resolvePath } from "@pixeloven/core";
 import { Plop } from "./plop";
 
 /**
- * Acceptable types
+ * Acceptable atomic types
  */
-const componentTypes = ["atom", "molecule", "organism", "page", "template"];
+const atomicTypes = ["atom", "molecule", "organism", "page", "template"];
```

Comment:
Let's add "partial" here

Reply:
Good call, already forgot about this new one.